### PR TITLE
chore(liveagent): add live chat css for MSPACE / Salesforce

### PR DESCRIPTION
**Pull request by gepifanio**
Ticket: https://jira.unimelb.edu.au/browse/TS-481

### CODE REVIEW

**Comment by gepifanio** [comment]
Comment: Deploy Started

Building with commit 69df8667f36b4b4dcdda364409d0cba78e2c32d5

https://app.netlify.com/sites/pattern-lib-unimelb/deploys/5d786b6d72d1c2000877b1cb

**Comment by kebfrf** [comment]
remove 
```
@font-face { 
    font-family: 'Salesforce Sans'; 
    src: url('//www.sfdcstatic.com/system/shared/common/assets/fonts/SalesforceSans/SalesforceSans-Regular.woff&#39;) format('woff'), 
    url('//www.sfdcstatic.com/system/shared/common/assets/fonts/SalesforceSans/SalesforceSans-Regular.ttf&#39;) format('truetype'); 
```

from code. apparently it's redundant and the font file won't appear any time soon

File: components/footer/index.css [hunk 1]
```diff
@@ -261,6 +261,23 @@
   width: 100%;
 }
 
+.embeddedServiceHelpButton .helpButton .uiButton {
```
Comment (TimMansell) [inline components/footer/index.css]: We should move this code to a third party section.

- [ ] migrate css to a file called `salesforce.css`
- [ ] save file in new top level folder called `/third-party`
- [ ] import into `/components/index.css` after `components`

File: components/index.css [hunk 2]
```diff
@@ -5,6 +5,7 @@
 @import '_shims.css';
 @import '_utils.css';
 @import '_grid.css';
+@import '../third-party/salesforce.css';
```
Comment (TimMansell) [inline components/index.css]: Can you move this import after the component imports

```
/* Third party - Alphabetically ordered */
@import ...
```

**Pull request merged**